jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/ContextHandlerFactory.java

```diff
@@ -40,16 +40,7 @@ public interface ContextHandlerFactory
      * The attribute name for the environment name.
      */
     String ENVIRONMENT_ATTRIBUTE = "environment";
-    /**
-     * Attributes that start with this name return a String pointing to XML on the filesystem that should be used for Environment based
-     * modifications to the ContextHandler that was created.
-     *
-     * <pre>
-     * jetty.deploy.environmentXml=config/base-ee8.xml
-     * jetty.deploy.environmentXml.auditing=config/auditing-ee8.xml
-     * </pre>
-     */
-    String ENVIRONMENT_XML_ATTRIBUTE = "jetty.deploy.environmentXml";
```

Comment:
Should we keep this constant as a deprecated for removal field and then do a check for any usage of it and emit a warning?   Or do we think it is unlikely that anybody used this as it was not documented?

Reply:
It wasn't documented, true.
The likelihood of it being used is extremely low.
This class is new for 12.1.x so I see no reason to immediately deprecate this private constant.

Now, that being said, the same String, used to exist in the `org.eclipse.jetty.server.Deployable` interface.
It is no longer there in 12.1.x.
Perhaps adding it back to `Deployable` and deprecating it makes sense.
But then I should really go through ALL of the constants in that `Deployable` class from 12.0.x and mark them deprecated (lemme do that in a different PR tho).

However, I can put in a warning if that string is used as a prefix in a properties file. That work for you?

Reply:
I went ahead and opened to address this more holistically.

* #13011